spec.html

```diff
@@ -10569,6 +10569,7 @@ <h1>OrdinaryCreateFromConstructor ( _constructor_, _intrinsicDefaultProto_ [ , _
       <emu-alg>
         1. Assert: _intrinsicDefaultProto_ is a String value that is this specification's name of an intrinsic object. The corresponding object must be an intrinsic that is intended to be used as the [[Prototype]] value of an object.
         1. Let _proto_ be ? GetPrototypeFromConstructor(_constructor_, _intrinsicDefaultProto_).
+        1. If _internalSlotsList_ is not present, set _internalSlotsList_ to a new empty List.
```

Comment:
The prose for this AO already says “ If internalSlotsList is not provided, a new empty List is used.”.

Reply:
While that's true, I think it's nicer to have it in the algorithm steps as well. There's a few other places we do this, like in [Abstract Relational Comparison](https://github.com/tc39/ecma262/pull/2378#discussion_r610333389), and I think we should change all of them.

Reply:
In that case, should it be removed from the prose?

Reply:
I think it's fine to keep it in the prose as well - it's an accurate and potentially useful description of the behavior of the algorithm.

Reply:
would you expect, then, that it'd be duplicated in prose and an algorithm step in every case where there's a default value on an optional AO argument?

Reply:
No, the preambles are just general descriptions of the behavior; they don't need to all give exactly the same level of details.